src/Form/createForm.tsx

```diff
@@ -0,0 +1,300 @@
+import * as React from "react";
+import {
+  EmitterSubscription,
+  findNodeHandle,
+  InteractionManager,
+  LayoutChangeEvent,
+  Keyboard,
+  ScrollView,
+  ScrollViewProps,
+  TextInput,
+  UIManager,
+  View,
+} from "react-native";
+
+interface FocusableContainer {
+  focusableRef: React.RefObject<any>;
+}
+
+interface FormRootState {
+  context: {
+    focusNext: (index: number) => void;
+    setFieldInstance: (
+      index: number,
+      focusable: FocusableContainer | null
+    ) => void;
+  };
+
+  scrollViewHeight: number;
+  scrollViewContentHeight: number;
+  containerMinHeight: number;
```

Comment:
Leave a comment to explain what purpose `containerMinHeight`serves.

Reply:
ok